pgcli/main.py

```diff
@@ -84,6 +86,14 @@
 
 from textwrap import dedent
 
+try:
+    import sshtunnel
```

Comment:
I made it optional because it is not a feature that most pgcli users will have the need for I think and it adds paramiko and sshtunnel as dependencies. But let me know if you prefer it wasn't optional.